### DISCUSSION TITLE
make changing background-image simple again

### DISCUSSION THREAD

**Discussion by jrabello** [regolith-linux/regolith-desktop#724] [Ideas]
Why changing background-image is SO HARD?

With Rigolith 1.6 we were able to change background image with settings app($mod+c)

Now with 2.0 everytime I change my background-image and restart my machine it comes back to the default background-image from darcula theme, please consider editing background-image feature simple again, this problem is very frustrating

**Comment by saltbr1nger** [comment]
Cosign.  New user here trying to figure out how to even get my designated wallpaper & lockscreen pictures to stick after a restart.

**Comment by abacao** [comment]
The current solution is to add the following lines to the Xresources file

```
regolith.wallpaper.file: 
regolith.wallpaper.color.primary: 
```

https://regolith-desktop.com/docs/using-regolith/configuration/#wallpaper

Regolith team just needs to add the "Background" option to the regolith configuration menu

**Comment by kgilmer** [comment]
> Why changing background-image is SO HARD?

In Regolith 1.x there were two was of setting wallpaper, via "dotfiles" and via gnome-settings.  A common problem is users would mix the two up and be puzzeled about why the wallpaper was changing unexpectedly.  To at least make the update process easy to understand, we opted to support dotfiles approach first.

> Regolith team just needs to add the "Background" option to the regolith configuration menu

Actually we're going to be re-adding the gnome-settings background panel back, but now it modifies the "dotfiles" to provide the consistent experience..

In any case, yeah sorry for the usability regression!  :peace_symbol: 

From my 2.2 dev env:

![image](https://user-images.githubusercontent.com/49683/198180251-6d943f49-5de2-4244-a5f5-f92912906bcc.png)
